.goreleaser.yml

```diff
@@ -19,13 +19,6 @@ builds:
   - -X github.com/projectcontour/contour-authserver/pkg/version.Version={{ .Env.VERSION }}
   - -X github.com/projectcontour/contour-authserver/pkg/version.Sha={{ .Env.SHA }}
   - -X github.com/projectcontour/contour-authserver/pkg/version.BuildDate={{ .Date }}
-archives:
-- replacements:
-    darwin: Darwin
-    linux: Linux
-    windows: Windows
-    386: i386
-    amd64: x86_64
```

Comment:
I don't think these are important to carry forward, it's easier to just use the names goreleaser gives us OOTB